Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -34,7 +34,7 @@ open class _ChatChannelListVC<ExtraData: ExtraDataTypes>: ViewController,
         return collection
     }()
     
-    public private(set) lazy var createNewChannelButton: ChatChannelCreateNewButton<ExtraData> = {
+    public private(set) lazy var createNewChannelButton: UIButton = {
         let button = uiConfig.channelList.newChannelButton.init()
         button.translatesAutoresizingMaskIntoConstraints = false
         button.addTarget(self, action: #selector(didTapCreateNewChannel), for: .touchUpInside)
```

Comment:
@VojtaStavik Because now uiConfig only knows that is an UIButton, here we need to set that it is a UIButton. If the client is customizing the button of channelListVC, or wants to populate extra data with it, isn't this an issue?

Reply:
Nope, we don't assign any data to if be the default. This is exactly how it should be 👍